### PR TITLE
Get user activities

### PR DESCRIPTION
**Pull request by luc779**

### CODE REVIEW

File: Utils/DataRetrieval.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+using MySqlConnector;
+
+namespace BackEndFucntions
+{
+    public class DataRetrieval
+    {
+        public List<dynamic> RetrieveCertainType(int userID, DateTime today, string ACTIVITY_TYPE)
+        {
+            List<dynamic> Transports = null;
+            string QUERY = "SELECT FROM Activities WHERE UserID = '" + userID + "' AND DateTime = '" + today.ToString("yyyy/MM/dd") + "' AND ActivityType = '" + ACTIVITY_TYPE + "'";
```
Comment (TseChapman) [inline Utils/DataRetrieval.cs]: `SELECT FROM` should be `SELECT Method, Amount FROM`

File: Utils/DataRetrieval.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
+using Company.Function;
+using MySqlConnector;
+
+namespace BackEndFucntions
+{
+    public class DataRetrieval
+    {
+        public List<dynamic> RetrieveCertainType(int userID, DateTime today, string ACTIVITY_TYPE)
+        {
+            List<dynamic> Transports = null;
```
Comment (TseChapman) [inline Utils/DataRetrieval.cs]: Use another variable name that fits more to this function.
For example like `activities`

File: Utils/DataRetrieval.cs [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+            string QUERY = "SELECT FROM Activities WHERE UserID = '" + userID + "' AND DateTime = '" + today.ToString("yyyy/MM/dd") + "' AND ActivityType = '" + ACTIVITY_TYPE + "'";
+
+            using MySqlConnection connetion = DatabaseConnecter.MySQLDatabase();
+            connetion.Open();
+
+            using MySqlCommand command = new(QUERY, connetion);
+
+            try {
+                command.CommandText = QUERY;
```
Comment (TseChapman) [inline Utils/DataRetrieval.cs]: Is this duplicated code to line 16?

File: Utils/DataRetrieval.cs (hunk 1 above)
Comment (TseChapman) [inline Utils/DataRetrieval.cs]: UserID is a foreign key, use a JOIN to validate userID exists in Users table. View GetEmissionHistory line 82 and 83 for example.

File: Utils/DataRetrieval.cs (hunk 3 above)
Comment (luc779) [reply Utils/DataRetrieval.cs]: yup

**Review by TseChapman — APPROVED** [review]

**Pull request merged**